lang/swi-prolog/Portfile

```diff
@@ -1,63 +1,64 @@
-PortSystem  1.0
-PortGroup   cmake 1.1
+PortSystem     1.0
```

Comment:
please add a [modeline](https://guide.macports.org/chunked/development.creating-portfile.html) as the first line followed by an empty line:
```
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

from that you'll see that the whitespace changes you made are not quite correct (they should be multiples of four). Additionally, we prefer non-functional changes in their own commit.